Homework3/Includes.h

```diff
@@ -0,0 +1,16 @@
+#ifndef INCLUDES.H
+#define INCLUDES .H
+
+#include <iostream>
+#include <vector>
+#include <string>
+using namespace std;
```

Comment:
do not pull all std namespace in your code. it pollutes it. use std::... instead